suite-common/logger/src/logsSlice.ts

```diff
@@ -0,0 +1,44 @@
+import { AnyAction, createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+export type LogEntry = { datetime: string; type: any; payload?: Record<any, any> };
+
+type LogsSliceState = {
+    logEntries: LogEntry[];
+};
+
+type LogsSliceRootState = {
+    logs: LogsSliceState;
+};
+
+const logsSliceInitialState: LogsSliceState = {
+    logEntries: [],
+};
+
+const MAX_ENTRIES = 200;
+
+export const logsSlice = createSlice({
+    name: 'logs',
+    initialState: logsSliceInitialState,
+    reducers: {
+        addLog: (
+            state,
+            action: PayloadAction<{
+                action: AnyAction;
+                payload: Record<string, unknown> | undefined;
+            }>,
+        ) => {
+            state.logEntries.push({
+                datetime: new Date().toUTCString(),
+                type: action.payload.action.type,
+                ...action.payload.payload,
```

Comment:
I'm sorry, my initial suggestion was wrong, we don't want to spread the payload here
```suggestion
                payload: action.payload.payload,
```

Reply:
It got simplified with the prepare method anyway https://github.com/trezor/trezor-suite/pull/7359/commits/2f69440b466e5668eede8d93f903fc5b7d445a18

Reply:
Resolved in https://github.com/trezor/trezor-suite/pull/7359/commits/f016a0e3c583900259ee8e1158482f057666193b as discussed during pair programming.